api.planx.uk/modules/gis/service/digitalLand.ts

```diff
@@ -68,6 +68,7 @@ async function go(
     entries: "current",
     geometry: geom,
     geometry_relation: "intersects",
+    exclude_field: "geometry,point",
```

Comment:
For a site that intersects with a few constraints, including complex ones like flood zones: 
- [Before example request](https://www.planning.data.gov.uk/entity.json?entries=current&geometry=MULTIPOLYGON+%28%28%28-0.12093+51.492328%2C+-0.120915+51.492299%2C+-0.11981+51.492079%2C+-0.119785+51.49208%2C+-0.119585+51.492034%2C+-0.119537+51.492132%2C+-0.119409+51.492204%2C+-0.12047+51.492409%2C+-0.120465+51.49242%2C+-0.120554+51.492439%2C+-0.120762+51.492512%2C+-0.12093+51.492328%29%29%29&geometry_relation=intersects&limit=100&dataset=article-4-direction-area&dataset=central-activities-zone&dataset=brownfield-land&dataset=brownfield-site&dataset=area-of-outstanding-natural-beauty&dataset=conservation-area&dataset=green-belt&dataset=national-park&dataset=world-heritage-site&dataset=world-heritage-site-buffer-zone&dataset=flood-risk-zone&dataset=listed-building&dataset=listed-building-outline&dataset=scheduled-monument&dataset=ancient-woodland&dataset=ramsar&dataset=special-area-of-conservation&dataset=special-protection-area&dataset=site-of-special-scientific-interest&dataset=park-and-garden&dataset=tree&dataset=tree-preservation-order&dataset=tree-preservation-zone) (response size ~3.7MB)
- [After example request](https://www.planning.data.gov.uk/entity.json?entries=current&geometry=MULTIPOLYGON+%28%28%28-0.12093+51.492328%2C+-0.120915+51.492299%2C+-0.11981+51.492079%2C+-0.119785+51.49208%2C+-0.119585+51.492034%2C+-0.119537+51.492132%2C+-0.119409+51.492204%2C+-0.12047+51.492409%2C+-0.120465+51.49242%2C+-0.120554+51.492439%2C+-0.120762+51.492512%2C+-0.12093+51.492328%29%29%29&geometry_relation=intersects&limit=100&dataset=article-4-direction-area&dataset=central-activities-zone&dataset=brownfield-land&dataset=brownfield-site&dataset=area-of-outstanding-natural-beauty&dataset=conservation-area&dataset=green-belt&dataset=national-park&dataset=world-heritage-site&dataset=world-heritage-site-buffer-zone&dataset=flood-risk-zone&dataset=listed-building&dataset=listed-building-outline&dataset=scheduled-monument&dataset=ancient-woodland&dataset=ramsar&dataset=special-area-of-conservation&dataset=special-protection-area&dataset=site-of-special-scientific-interest&dataset=park-and-garden&dataset=tree&dataset=tree-preservation-order&dataset=tree-preservation-zone&exclude_field=geometry,point) (response size ~2.5KB)